### ISSUE TITLE
Screen reader does not announce label information for 'Servers' combo box in windows.

### ISSUE THREAD

**Issue by msftedad** [swagger-api/swagger-editor#5563]
**Test Environment:**
OS: Windows 11 Enterprise 24H2 (OS build: 22621.2428)
Browser: Edge (Version 139.0.3405.102), Chrome Version 131.0.6778.206 (Official Build) (64-bit)
Screen Reader: NVDA (2025.1.2), JAWS (2025.2505.43), Narrator
URL: https://editor.swagger.io/

**Pre-Requisite:**

1. Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Narrator: Ctrl + win + Enter
2. Verbosity:
NVDA: Default
JAWS: Beginner, Highest
Narrator: Default

**User Impact:**
Screen reader users may find it difficult to understand the purpose of the combo box if screen reader does not announce label information for 'Servers' combo box.

**Repro Steps:**

1. Open the URL: https://editor.swagger.io/ in Edge browser in windows.
2. 'Swagger Editor' page will open.
3. Press tab key to navigate to the 'Servers' combo box.
4. Verify whether screen reader announces the label information for 'Servers' combo box in windows or not.

**Actual Result:**
Screen reader does not announce label information for 'Servers' combo box in windows.
Screen reader announces as: combo box https://petstore3.swagger.io/api/v3 collapsed

**Refer Attachment:**

1. Screen reader does not announce label information for 'Servers' combo box in windows.png
2. Screen reader does not announce label information for 'Servers' combo box in windows.mp4

<img width="972" height="520" alt="Image" src="https://github.com/user-attachments/assets/7e0eca78-5639-4997-b567-dbc71e944af3" />
https://github.com/user-attachments/assets/53dd3231-868e-426b-90e4-0bf54b702575

**Expected Result:**
Screen reader should announce label information for 'Servers' combo box in windows.
Screen reader should announce: Servers combo box https://petstore3.swagger.io/api/v3 collapsed

**Comment by robert-hebel-sb** [comment]
Hey @msftedad, thanks for creating this issue

Could you check whether the same issue exists on https://editor-next.swagger.io/, which utilizes the newest version of swagger-ui? If it still exists, please create an issue here: https://github.com/swagger-api/swagger-ui/issues and we will try to address it as soon as possible.
